DiabloII.Application.Tests/Suggestions/Comment/CommentASuggestionSteps.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using DiabloII.Application.Requests.Suggestions;
+using DiabloII.Application.Tests.Extensions;
+using DiabloII.Application.Tests.Startup;
+using DiabloII.Infrastructure.DbContext.Mappers.Suggestions;
+using TechTalk.SpecFlow;
+using TechTalk.SpecFlow.Assist;
+
+namespace DiabloII.Application.Tests.Suggestions.Comment
+{
+    [Binding]
+    [Scope(Tag = "suggestion")]
+    public class CommentASuggestionSteps
+    {
+        private readonly SuggestionApi _suggestionApi;
+        private readonly SuggestionTestContext _suggestionContext;
+
+        public CommentASuggestionSteps(MyTestContext testContext, SuggestionTestContext suggestionContext)
+        {
+            _suggestionApi = testContext.Apis.Suggestions;
+            _suggestionContext = suggestionContext;
+        }
```

Comment:
my DIP is bad, we should be able to send SuggestionAPI directly instead of using MyTestContext.